Src/LinearSolvers/MLMG/AMReX_MLTensorOp.cpp

```diff
@@ -247,20 +254,65 @@ MLTensorOp::apply (int amrlev, int mglev, MultiFab& out, MultiFab& in, BCMode bc
                          Array4<Real> const fyfab = fluxfab_tmp[1].array();,
                          Array4<Real> const fzfab = fluxfab_tmp[2].array(););
 
-            AMREX_LAUNCH_HOST_DEVICE_LAMBDA_DIM
```

Comment:
Could `fluxfab_tmp` be allocated in The_Async_Arena ? (slightly orthogonal comment)

Reply:
Yes.

Reply:
If we use The_Async_Arena, we will have to move `FArrayBox fluxfab_tmp[AMREX_SPACEDIM` into the MFIter loop body, because `resize` does not work in this situation.  The reason that resize works with Elixir is because Elxiir takes the ownership of the memory.  But currently Elixir does not work with The_Async_Arena.  We could make it work in the future.  I think right now we can just leave it like this.  For OpenMP, we do want `FArrayBox fluxfab_tmp` to be outside the MFIter loop so that they can be reused.

Reply:
Sounds good, thanks for checking !